backend/app/floorplans.go

```diff
@@ -0,0 +1,39 @@
+package app
+
+import (
+	"github.com/lanets/floorplanets/backend/models"
+)
+
+func (app *App) CreateFloorplan(name string) (*models.Floorplan, error) {
+	floorplan := &models.Floorplan{
+		Name: name,
+	}
+
+	err := app.Database.Create(floorplan).Error
+	if err != nil {
+		return nil, err
+	}
+
+	return floorplan, nil
+}
+
+func (app *App) GetFloorplan(id int) (*models.Floorplan, error) {
+	var floorplan models.Floorplan
+	if app.Database.First(&floorplan, id).RecordNotFound() {
+		return nil, app.Database.Error
+	}
+	return &floorplan, app.Database.Error
+}
+
+func (app *App) GetAllFloorplansNameId() ([]models.Floorplan, error) {
```

Comment:
> f we need to include more informations in the future

If we do, we will still need this method to exist.

the /florplans endpoint can't start dumping full-on floorplans. This would mean querying /floorplan can return big ass responses.